vizro-core/src/vizro/models/_navigation/_navigation_utils.py

```diff
@@ -8,14 +8,17 @@ def _validate_pages(pages):
     from vizro.models import Page
 
     pages_as_list = list(itertools.chain(*pages.values())) if isinstance(pages, dict) else pages
-
-    if not pages_as_list:
-        raise ValueError("Ensure this value has at least 1 item.")
-
     # Ideally we would use dash.page_registry or maybe dashboard.pages here, but we only register pages in
     # dashboard.pre_build and model manager cannot find a Dashboard at validation time.
     # page[0] gives the page model ID.
     registered_pages = [page[0] for page in model_manager._items_with_type(Page)]
+
+    if pages is None:
+        return registered_pages
```

Comment:
Could you add a comment here that this is required to auto-populate the `navigation.selector.pages` if this is actually true? I am currently assuming you had to add this to get all the configurations working?